designs/administrator/olm/list-of-operators/list-of-operators.md

```diff
@@ -0,0 +1,51 @@
+---
+parent: Administrator
+version: 4.5
+---
+
+# List of Operators
+
+## All projects view
+This is the “all-projects” view which has been modified to now expose more information about the namespaces and without listing out every CSV. The goal of this view is to consolidate the list of operators. There are a few additions to the flow:
+
+1. We have introduced the Operator Namespace column. The Operator Namespace column has 2 descriptors that can live in each row; the first is a single project like “my-project” that is associated with an operator like 3scale. If a namespace appears across All Namespaces it will appear as All Namespaces. In the example of an operator like Jaeger, the text underneath has the namespace name and All Namespaces underneath.
```

Comment:
So the Operator Namespace column can also say "all namespaces"? Just making sure I'm understanding correctly since in the mockup we only show "all namespaces" as an option for the Managed column.

Reply:
This is no longer true. I'll update this.